python_modules/dagit/dagit/starlette.py

```diff
@@ -180,6 +208,10 @@ def build_routes(self):
                 "/download_debug/{run_id:str}",
                 self.download_debug_file_endpoint,
             ),
+            Route(
+                "/dagit/notebook",
+                self.download_notebook,
+            ),
```

Comment:
the status quo dagit server has a `include_notebook_route` toggle because we'd like the default (e.g. dagster cloud) to disable "preview notebook". the reason of disabling the preview in cloud was the parsed html (from executable ipynb) could return malicious javascript. i'd assume we'd have the similar concern with the download option too.

see: https://github.com/dagster-io/dagster/blob/master/python_modules/dagit/dagit/app.py#L198
cc @gibsondan

Reply:
ah i misread this route. i think this _is_ the preview one, then we probably should disable it by default same as status quo

Reply:
we are using a different composition method for how endpoints are re-used (or omitted) in other webservers, so im not sure how critical supporting this flag is